var/spack/repos/builtin/packages/py-pygments/package.py

```diff
@@ -23,6 +23,7 @@
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
 from spack import *
+import os
```

Comment:
Sounds good. Not sure what it could have been; deptypes doesn't care much about the `os` module at all.